src/utils.js

```diff
@@ -158,3 +158,16 @@ export function sorter ($sort) {
     return comparator;
   };
 }
+
+export function makeUrl (path, app = {}) {
+  const get = typeof app.get === 'function' ? app.get : () => {};
+  const env = get('env') || process.env.NODE_ENV;
+  const host = get('host') || process.env.HOST_NAME || 'localhost';
+  const protocol = (env === 'development' || env === 'test' || (env === undefined)) ? 'http' : 'https';
```

Comment:
Right. My thought is that even if you have a proxy in front you *need* to be using https. Worst case in the off chance you you need to hack around that you could set your env to `development` or something.